migrate/20240711_add_severity_to_page.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:page) do
+      add_column :severity, :text, collate: '"C"', default: "error"
```

Comment:
should we create an enum that only allows these values: `critical` , `error`, `warning`, `info`?

Reply:
Makes sense. Added